stl/inc/format

```diff
@@ -1127,11 +1162,9 @@ _OutputIt _Write(_OutputIt _Out, const _CharT* _Value) {
     return _Out;
 }
 
-template <class _CharT, class _OutputIt, class _Ty>
-_OutputIt _Write(_OutputIt _Out, _Ty _Val) {
-    (void) _Val;
-    _STL_INTERNAL_CHECK(false);
-    return _Out;
+template <class _CharT, class _OutputIt>
+_OutputIt _Write(_OutputIt _Out, basic_string_view<_CharT> _Value) {

```

Comment:
Should this handle `basic_string_view<_CharT, _Traits>`?

Reply:
I *think* this takes `_Value` from the ``basic_format_args` store, so at this point everything should be in `basic_string_view<_CharT>` form

Reply:
`_Value` comes from `basic_format_args`, as you've stated, so I am only handling the values available from that.